docs/api/collections.rst

```diff
@@ -5,100 +5,172 @@ Collections
 
 A collection belongs to a bucket and stores records.
 
-A collection is a mapping with the following attributes:
+A collection is a mapping with the following attribute:
 
 * ``permissions``: (*optional*) the :term:`ACLs <ACL>` for the collection object
 
-
 .. .. note::
 
 ..     By default users have a bucket that is used for their own data.
-..     Application can use this default bucket with the ``~`` shortcut.
-..     ie: ``/buckets/~/collections/contacts`` will be the current user contacts.
+..     Application can use this default bucket with the ``default`` shortcut.
+..     ie: ``/buckets/default/collections/contacts`` will be the current
+..     user contacts.
+
+
+Creating a collection
+=====================
+
 
+.. http:put:: /buckets/(bucket_id)/collections/(collection_id)
 
-PUT /buckets/<bucket_id>/collections/<collection_id>
-====================================================
+    **Requires authentication**
+    Creates or replaces a collection object.
```

Comment:
I would add something like for buckets:

A collection is the parent object of records. It can be viewed as a container where records permissions are assigned globally.